### ISSUE TITLE
Toolbar icons are not actionnable

### ISSUE THREAD

**Issue by fbessou** [aforemny/elm-mdc#50]
According to https://material.io/components/web/catalog/toolbar/ toolbar icons should be using `<a>`, `<span>` or `<button>` tags.
The current implementation in elm-mdc is using `<i>` which does not allow to use toolbar icons as a links.

**Comment by aforemny** [comment]
Hi @fbessou, thanks for reporting!

I like the following that I have come up with and pushed it to master:

`Toolbar.icon` and `Toolbar.menuIcon` are Icon properties instead of HTML elements.

```
icon : Icon.Property m
menuIcon : Icon.Property m
```

and you would use `Icon.view` like so:

```
Icon.view [ Icon.anchor, Toolbar.icon ] "bookmark"
```

I added Icon properties to modify the HTML element of the `Icon.view`, there is `Icon.anchor`, `Icon.button` and `Icon.span`.

**Comment by fbessou** [comment]
Hi @aforemny, thank you very much for this feature. :D
I really like the way it works :)

**Comment by aforemny** [comment]
@fbessou I made a typo in `Icon.anchor`. It is fixed in recent master (17f77af9124c480576bb0e423acd08f4899236c3).

**Comment by fbessou** [comment]
Thanks :)
